reposerver/repository/repository.go

```diff
@@ -486,8 +486,12 @@ func findManifests(appPath string, env *v1alpha1.Env, directory v1alpha1.Applica
 			objs = append(objs, &obj)
 		} else if strings.HasSuffix(f.Name(), ".jsonnet") {
 			vm := makeJsonnetVm(directory.Jsonnet, env)
+			jpaths := []string{appPath}
+			for _, p := range directory.Jsonnet.JPaths {
+				jpaths = append(jpaths, filepath.Join(appPath, p))
+			}
 			vm.Importer(&jsonnet.FileImporter{
-				JPaths: []string{appPath},
+				JPaths: jpaths,
```

Comment:
It should be documented somewhere that `appPath` will be an implicit member of `jpath`, even if not explicitly specified by `--jsonnet-jpath`